datasources/automatic/DATASOURCE_INSTRUCTIONS.md

```diff
@@ -0,0 +1,8 @@
+# Datasources
+
+## How to add sources
+1. Please add official datasources of any language to the sources.csv file
```

Comment:
I think we need some more explanation about the idea of the "smart scraper". Either here or in the issue that you create. It must be clearer, that the scrapers main job is to find pairs of question + answer within some html code.
The rest (scheduling, ingestion into ES, filling in metadata in the CSV) is all nice but could be handled independently of the "smart scraper".

Reply:
Agreed, I give more info on the smart scraper in the issue and only link this for data addition.